gitpod-io/gitpod
# Fix colon in URL params breaks Gitpod extension

## Description



## Related Issue(s)

Fixes #6514

## How to test

1. Go to https://jp-fix-6514.staging.gitpod-dev.com/#TESTING_URL_PARAMS_WITH_COLON=:/https://github.com/shaal/testing--gitpod-url-params-with_colon
2. Check that the gitpod extension is working properly

## Release Notes

```release-note
NONE
```

## Documentation
